easybuild/easyblocks/l/llvm.py

```diff
@@ -113,4 +113,16 @@ def configure_step(self):
             else:
                 raise EasyBuildError("Failed to find unpacked CMake modules directory at %s", cmake_modules_path)
 
+        if LooseVersion(self.version) >= LooseVersion('16.0'):
+            # make sure that third-party modules are available in build directory,
+            # and if so make a 'third-party' symlink so LLVM can find them
+            third_party_modules_path = os.path.join(self.builddir, 'third-party-%s.src' % self.version)
+            if os.path.exists(third_party_modules_path):
+                cwd = change_dir(self.builddir)
+                symlink('third-party-%s.src' % self.version, 'third-party')
+                change_dir(cwd)
```

Comment:
2 Questions here: Why the symlink and not a move? The name "third-party-version.src" is only because of the filename of the archive, I don't see how it is required to be kept.

2nd: Why the `change_dir`? `symlink('third-party-%s.src' % self.version, os.path.join(self.builddir, 'third-party'), use_abspath_source=False)` would be better, wouldn't it?   
But I'd just rename it instead.

Reply:
@Flamefire If you think it's worth changing this, please open an issue or a PR, following up on stuff in merged PRs is a nightmare I don't want to get into :)

Reply:
I see: It was C&P from the cmake folder above. Opened PR: https://github.com/easybuilders/easybuild-easyblocks/pull/2994